pkgs/development/ocaml-modules/cduce/common.nix

```
@@ -0,0 +1,14 @@
+{ lib }:
+
+{
+  version = "0.7.0-rc11";
+  useDune2 = true;
+  src = builtins.fetchGit {
+    url = "https://gitlab.math.univ-paris-diderot.fr/cduce/cduce.git";
+    rev = "bcbb652879090fe0a0531491d8048296a343cbcf";
```

I choosed this commit because it is when they bumped the rc number we could go unstable with latest master and we compile up to 4.13. I let the requester to choose.
```suggestion
    rev = "3cba0c56c55a553bc2e4d1f23baf14096d156d0c";
```